server/app/models/host_node.rb

```diff
@@ -68,7 +71,6 @@ def to_path
   # @param [Hash] attrs
   def attributes_from_docker(attrs)
     self.attributes = {
-      node_id: attrs['ID'],
```

Comment:
I'm not entirely sure what this mechanism to update the `HostNode.node_id` in the `/nodes/update` RPC is meant for, but I feel like it does more harm than good. The node ID shouldn't ever change.